packages/store-ui/src/atoms/Spinner/Spinner.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { forwardRef } from 'react'
+
+export type SpinnerProps = {
+  /**
+   * ID to find this component in testing tools (e.g.: cypress, testing library, and jest).
+   */
+  testId?: string
+}
+
+const Spinner = forwardRef<HTMLDivElement, SpinnerProps>(function Spinner(
+  { children, testId = 'store-spinner', ...props },
+  ref
+) {
+  return (
+    <span ref={ref} data-store-spinner data-testid={testId} {...props}>
+      {children}
+    </span>
```

Comment:
What do you think about this suggestion?
```suggestion
    <span ref={ref} data-store-spinner data-testid={testId} {...props} />
```